docs/install.md

```diff
@@ -5,33 +5,32 @@ post installation to ensure the operator can function correctly.
 
 ## STS Clusters
 
-In an STS Cluster, *CredentialsRequests* are not automatically provisioned by
-the `cloud-credential-operator` and requires manual intervention by the
-cluster-admin. The credentials secret needs to be provisioned using the `ccoctl` binary.
+### Post operator installation
 
-The `aws-load-balancer-operator` also relies on the `cloud-credential-operator`
-to provision the secret for *CredentialsRequest*. And so in an STS Cluster the
-secret needs to be provisioned manually. The operator will wait until the required
-secrets are created and available.
+In an STS Cluster, `CredentialsRequest`s are not automatically provisioned by
+the **cloud-credential-operator** and the manual intervention done by the
+cluster-admin is required. IAM role and policies as well as the credentials secret need to be provisioned manually for the further consumption by the PODs.
+`ccoctl` binary can be used to facilitate this task.
 
-### Pre-Requisites
+The **aws-load-balancer-operator** also relies on the **cloud-credential-operator**
+to provision the secret for the operated controller using `CredentialsRequest`. And so in an STS Cluster the
+secret needs to be provisioned manually. The **aws-load-balancer-operator** will wait until the required
+secret is created and available before spawning **aws-load-balancer-controller** POD.
 
-### [Extract and prepare the `ccoctl` binary](https://docs.openshift.com/container-platform/4.10/authentication/managing_cloud_provider_credentials/cco-mode-sts.html#cco-ccoctl-configuring_cco-mode-sts)
+#### Pre-Requisites
 
-### Extract required `CredentialsRequests`
+#### [Extract and prepare the `ccoctl` binary](https://docs.openshift.com/container-platform/4.11/authentication/managing_cloud_provider_credentials/cco-mode-sts.html#cco-ccoctl-configuring_cco-mode-sts)
 
-1. The **aws-load-balancer-operator** creates a *CredentialsRequest* named
-`aws-load-balancer-controller-<cr-name>` in the *openshift-cloud-credential-operator* namespace.
+#### Extract required `CredentialsRequests`
+
+1. For each `AWSLoadBalancerController` CR the **aws-load-balancer-operator** creates a `CredentialsRequest` named `aws-load-balancer-controller-<cr-name>` in the `openshift-cloud-credential-operator` namespace. Extract and save the created `CredentialsRequest` in a directory:
```

Comment:
Only a single instance of the `AWSLoadBalancerController` can be created since it will only reconcile CR's with the name cluster. See https://github.com/openshift/aws-load-balancer-operator/blob/main/docs/tutorial.md#post-installation

Reply:
Yes, I remembered this when updated the doc. The decision to have a single instance was made due to a technical limitation not fixed upstream, there was no intent to limit this in ALBO. One day the technical limitation will be removed and multiple ALBC will be allowed.
However I agree, even though this phrase is technically correct it may mislead the reader.